net/openssh/Portfile

```diff
@@ -54,8 +54,10 @@ if {${name} eq ${subport}} {
                         pam.patch \
                         patch-sandbox-darwin.c-apple-sandbox-named-external.diff \
                         patch-sshd.c-apple-sandbox-named-external.diff \
-                        macports-config.patch
+                        macports-config.patch \
+                        patch.zlib1.3_configure.ac9.4pre2.diff
 
+    # the last patch above is a fix for zlib 1.3 from upstream before next release: https://github.com/openssh/openssh-portable/commit/cb4ed12ffc332d1f72d054ed92655b5f1c38f621
```

Comment:
Usually it's simpler if this comment goes in the patchfile itself.

Reply:
I tried that approach, see other comment. I will try it again.